src/ChunkReader.php

```diff
@@ -84,7 +83,8 @@ public function read(WithChunkReading $import, Reader $reader, TemporaryFile $te
         $afterImportJob = new AfterImportJob($import, $reader);
 
         if ($import instanceof ShouldQueueWithoutChain) {
-            $jobs->push($afterImportJob->delay($delayCleanup));
+            $afterImportJob->setDependencies($jobs);
+            $jobs->push($afterImportJob->delay(60));
```

Comment:
why remove the `$delayCleanup`? I think it should be changeable from userland

Reply:
Because it was just added as a workaround to run the cleanup only after an hour or so, with this PR it becomes unnecessary and would actually interfer with the fix if people forget to remove it, so if we leave the option to change the interval, it should be a differently named option

Reply:
Yeah...

Reply:
I think we should give it another name so it can be changed instead of a constant delay of `60`